### ISSUE TITLE
OAuthException issue

### ISSUE THREAD

**Issue by Amraneze** [Amraneze/react-instagram-login#1]
Getting this response
````json
{
"error_type": "OAuthException",
"code": 400,
"error_message": "Invalid scope: []"
}
````
After trying to login.

Reported by @akshayF12

**Comment by Amraneze** [comment]
@akshayF12 do you have an example ? I created this [sandbox](https://codesandbox.io/s/amraneze-react-instagram-login-gggjr?file=/src/App.js:73-104) if you can reproduce it with it. (I don't have an instagram account anymore :( )

**Comment by Amraneze** [comment]
@akshayF12 I released a new version [3.0.0](https://www.npmjs.com/package/@amraneze/react-instagram-login) but it doesn't contain the render function anymore after moving to React Hooks. Can you test it and let me know

**Comment by akshayF12** [comment]
@Amraneze Hello thanks for the replied i have tested new version got  authentication code successfully generated but only for tester user which we have added on basic Instagram setting on Facebook dev,  when we generated  for new user got  "error message": "Invalid scope: []" don't know why this happened any solutions for this???

**Comment by Amraneze** [comment]
@akshayF12  which scope are you adding to the props of the component ? or are you using just the default one which is `user_profile` ?

**Comment by akshayF12** [comment]
@Amraneze scope i am using is user_media,user_profile

**Comment by Amraneze** [comment]
> @Amraneze scope i am using is user_media,user_profile

I will try to test with those scopes and I will let you know.

**Comment by brawll** [comment]
@Amraneze any updates on this, can we pass user_media in the scope now?

**Comment by Amraneze** [comment]
Hi @brawll, I will try to add this feature during this weekend

**Comment by Amraneze** [comment]
@brawll I pushed a new commit which contains:
- Allowing to pass the scope in props
- Make demo to run smoothly with the command `yarn demo`

Let me know if that fixes the issue for you.

>PS: I updated also the [sandbox](https://codesandbox.io/s/amraneze-react-instagram-login-gggjr) with the latest version `3.0.4` if you want to test in live

**Comment by Amraneze** [comment]
@brawll there was a bug with Instagram API url. I fixed in the version `3.0.5`. If it's ok for you, I can close this issue ?

**Comment by brawll** [comment]
Yeah, man you were a great help things worked mostly fine for me even at version 3.0.4
Thanks for the quick response & fixing.🙏
